### PR TITLE
DOC: Update pandas.core.groupby.GroupBy.pipe docstring

### PR DESCRIPTION
**Pull request by hsjsjsj009**
- [x] closes https://github.com/pandanistas/pandanistas_sprint_ui2020/issues/2
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/groupby/groupby.py [hunk 1]
```diff
@@ -190,14 +190,14 @@ class providing the base-class of operations.
...
-Apply a function `func` with arguments to this %(klass)s object and return
-the function's result.
+Apply a function `func` with arguments to this %(klass)s object and return the result.
 
 %(versionadded)s
 
-Use `.pipe` when you want to improve readability by chaining together
-functions that expect Series, DataFrames, GroupBy or Resampler objects.
-Instead of writing
+Use `.pipe` when you want to improve readability.
```
Comment (WillAyd) [inline pandas/core/groupby/groupby.py]: Can you revert the changes to this paragraph?

File: pandas/core/groupby/groupby.py [hunk 2]
```diff
@@ -212,19 +212,20 @@ class providing the base-class of operations.
 
 Parameters
 ----------
-func : callable or tuple of (callable, string)
-    Function to apply to this %(klass)s object or, alternatively,
+func : callable or tuple of (callable, str)
+    Function to apply to this %(klass)s object or, alternatiy,
```
Comment (WillAyd) [inline pandas/core/groupby/groupby.py]:
```suggestion
    Function to apply to this %(klass)s object or
```

**Review by WillAyd — CHANGES_REQUESTED** [review]
looks good! a few edits

**Comment by datapythonista** [comment]
@hsjsjsj009 do you have time to address the comments? Would be great to get this merged.

**Comment by hsjsjsj009** [comment]
Sorry, I just saw this email, I will do it today.

File: pandas/core/groupby/groupby.py [hunk 3]
```diff
@@ -190,14 +190,13 @@ class providing the base-class of operations.
...
-Apply a function `func` with arguments to this %(klass)s object and return
-the function's result.
+Apply a function `func` with arguments to this %(klass)s object and return the result.
 
 %(versionadded)s
 
 Use `.pipe` when you want to improve readability by chaining together
-functions that expect Series, DataFrames, GroupBy or Resampler objects.
-Instead of writing
+functions expect Series, DataFrames, GroupBy or Resampler objects.
```
Comment (datapythonista) [inline pandas/core/groupby/groupby.py]: can you restore `that`

**Review by datapythonista — COMMENTED** [review]
Looks good, just a small thing.

File: pandas/core/groupby/groupby.py (hunk 3 above)
Comment (hsjsjsj009) [reply pandas/core/groupby/groupby.py]: okay

**Pull request closed without merging**